### PR TITLE
Homework #5: Polindrom

### PR DESCRIPTION
**Pull request by AnastasiiaDom**

### CODE REVIEW

File: homework/anastasiadom/nastia_hw5.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import re
+import sys
+
+def StrPolindrom(s):
```
Comment (aodarc) [inline homework/anastasiadom/nastia_hw5.py]: fix function name format

File: homework/anastasiadom/nastia_hw5.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+	s1=s.lower()
```
Comment (aodarc) [inline homework/anastasiadom/nastia_hw5.py]: you do not need 3 variables make it in one line

File: homework/anastasiadom/nastia_hw5.py (hunk 1 above)
Comment (AnastasiiaDom) [reply homework/anastasiadom/nastia_hw5.py]: done

File: homework/anastasiadom/nastia_hw5.py (hunk 2 above)
Comment (AnastasiiaDom) [reply homework/anastasiadom/nastia_hw5.py]: done

**Pull request merged**